oclc/classify.py

```diff
@@ -795,10 +1047,10 @@ class TitleAuthorLookupCoverageProvider(IdentifierCoverageProvider):
     SERVICE_NAME = "OCLC Classify Coverage Provider"
     INPUT_IDENTIFIER_TYPES = [Identifier.GUTENBERG_ID, Identifier.URI]
     DATA_SOURCE_NAME = DataSource.OCLC
-    
+
     def __init__(self, _db, api=None, **kwargs):
         super(TitleAuthorLookupCoverageProvider, self).__init__(
-            _db, registered_only=True, **kwargs
+            _db, **kwargs
```

Comment:
`registered_only=True` still needs to be present. Otherwise we'll go around trying to do a title/author lookup on identifiers that aren't actually being used.